gravity.sh

```diff
@@ -33,13 +33,17 @@ regexFile="${piholeDir}/regex.list"
 adList="${piholeDir}/gravity.list"
 blackList="${piholeDir}/black.list"
 localList="${piholeDir}/local.list"
+whiteList="${piholeDir}/white.list"
 VPNList="/etc/openvpn/ipp.txt"
 
+piholeGitDir="/etc/.pihole"
+gravityDBfile="${piholeDir}/gravity.db"
+gravityDBschema="${piholeGitDir}/advanced/Templates/gravity.db.schema"
+optimize_database=false
```

Comment:
Why is optimizing the database not on by default?

Reply:
It can take a *very long* time for those who run > 1 mio. domains. @PromoFaux tested it and it was no joy.